User/Makefile

```diff
@@ -101,8 +101,8 @@ ifeq ($(FUZZ_MEM),)
 endif
 
 ifeq ($(COVERAGE),1)
-	CFLAGS  += -fprofile-arcs -ftest-coverage -D COVERAGE_TEST
-	LDFLAGS += --coverage
```

Comment:
This will not work on macOS, can we just not add `-Wl,-wrap,llvm_gcda_emit_arcs` for macOS?